src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -17,15 +17,15 @@ public Object fooString()
         String fakeUnicode = "asd\tsasd"; 
         String fakeUnicode2 = "\\u23\\u123i\\u";
         String content = "";
-        return "\ufeff" + content ; // byte order mark ok
+        /*byte order mark ok*/return "\ufeff" + content ;
```

Comment:
Just to confirm, this change is because if `feff` is marked as bad, the test won't trigger a violation on this line because it has a comment afterwards, but with a comment before, it will?
If so, then I think it is fine.